### ISSUE TITLE
Problem with negative integers

### ISSUE THREAD

**Issue by wavewizard** [florinpatrascu/bolt_sips#42]
Hi Florin, thank you for  the efforts you are putting in to this project.

Today I`ve seen a strange behaviour when using bolt_sips driver.  It somewhat gives wrong result when there are  negative integers on neo4j. for example: an integer which value is -1 become 256 and  -256 becomes 65280 in bolt_sips query result. 

It seems bolt_sips does not interpret  neo4j`s negative integers.

**Comment by florinpatrascu** [comment]
@wavewizard - thank you for the bug report! 

@mschae - would you please double check Boltex, the `Boltex.PackStream.decode/1`, in case you can spare some time? Boltex returns `255` for `-1`. Simple test:

```elixir
iex> Boltex.test('localhost',7687, "return -1 as n")
[success: %{"fields" => ["n"], "result_available_after" => 3}, record: [255],
 success: %{"result_consumed_after" => 0, "type" => "r"}]
```
The encoder is correct.

Thank you both!

**Comment by mschae** [comment]
Wow this completely slipped by, sorry.

Will add negative integers asap, sorry about that

**Comment by mschae** [comment]
@florinpatrascu Can you verify that my latest master fixes negative integers? If that's so I'll release a new version. Sorry for the delay.

**Comment by florinpatrascu** [comment]
yep, the `master` branch is the fix :) Thank you, @mschae! I'll push a new release as soon release you have time to publish the master.

**Comment by mschae** [comment]
@florinpatrascu Just published 0.4.1 with the fix. Thanks for the report!

**Comment by florinpatrascu** [comment]
Thank you both, @mschae and @wavewizard! Negative numbers are in, now =) 

@wavewizard - please use the latest `bolt_sips` version, from the `master` branch.
